examples/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.10.2 FATAL_ERROR)
```

Comment:
@azeey do you think this should be reverted as well?

Reply:
I think so for the same reason as https://github.com/gazebosim/sdformat/pull/1413#discussion_r1598975231